app/controllers/hyku/invitations_controller.rb

```diff
@@ -26,7 +26,7 @@ def create
       yield resource if block_given?
 
       # Override destination as this was a success either way
-      if is_flashing_format? && resource.invitation_sent_at
+      if is_flashing_format? && resource.invitation_sent_at # rubocop:disable Style/IfUnlessModifier
```

Comment:
Still confused as to why this change is being made. It's not part of the theming work.

Reply:
Because this is already in Hyku without the disable comment, I would remove it.